lib/util/Timer.js

```diff
@@ -1,10 +1,15 @@
 var _DEBUG = typeof DEBUG === 'undefined' ? false : DEBUG;
 var _ASSERT = typeof ASSERT === 'undefined' ? false : ASSERT;
 
-module.exports.now = function() {
+export const now = function() {
   return Date.now();
-}
+};
 
-module.exports.diff = function(time) {
+export const diff = function(time) {
   return Date.now() - time;
+};
+
+export default {
```

Comment:
I don't think this default export is necessary. If we want to import both functions we can use `import * as Timer from "Timer.js"` (same in common.js)